include/emitc/emitc_tosa.h

```diff
@@ -49,6 +49,72 @@ inline Src add(Src x, Src y) {
   return emitc::add<Src>(x, y);
 }
 
+// MulOp
+template <typename Src>
+inline Src mul(Src x, Src y, int32_t shift) {
```

Comment:
Since you have written a `MulOpConversion` I suggest to drop `int32_t shift` and the assert completely: the compiler will be forced to choose the right function or fail miserable :)

Reply:
I commited your suggestion.